src/Compilers/CSharp/Test/Symbol/DocumentationComments/CrefTests.cs

```diff
@@ -278,25 +278,25 @@ class Program { }
             compilation.VerifyDiagnostics(
                 // (4,20): warning CS1584: XML comment has syntactically incorrect cref attribute ':'
                 // /// See <see cref=":"/> - first character is colon.
-                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, ":").WithArguments(":"),
+                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, ":").WithArguments(":").WithLocation(4, 20),
                 // (4,20): warning CS1658: Identifier expected. See also error CS1001.
                 // /// See <see cref=":"/> - first character is colon.
-                Diagnostic(ErrorCode.WRN_ErrorOverride, ":").WithArguments("Identifier expected", "1001"),
+                Diagnostic(ErrorCode.WRN_ErrorOverride, ":").WithArguments("Identifier expected", "1001").WithLocation(4, 20),
                 // (5,20): warning CS1584: XML comment has syntactically incorrect cref attribute '::'
                 // /// See <see cref="::"/> - first character is colon.
-                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, ":").WithArguments("::"),
+                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, "::").WithArguments("::").WithLocation(5, 20),
                 // (5,20): warning CS1658: Identifier expected. See also error CS1001.
                 // /// See <see cref="::"/> - first character is colon.
-                Diagnostic(ErrorCode.WRN_ErrorOverride, "::").WithArguments("Identifier expected", "1001"),
+                Diagnostic(ErrorCode.WRN_ErrorOverride, "::").WithArguments("Identifier expected", "1001").WithLocation(5, 20),
                 // (6,20): warning CS1584: XML comment has syntactically incorrect cref attribute '&#58;&#58;Gibberish'
                 // /// See <see cref="&#58;&#58;Gibberish"/> - first character is colon.
-                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, "&").WithArguments("&#58;&#58;Gibberish"),
+                Diagnostic(ErrorCode.WRN_BadXMLRefSyntax, "&#58;&#58;").WithArguments("&#58;&#58;Gibberish").WithLocation(6, 20),
```

Comment:
the offset/width was pre ious being determined by the *next* token (not hte skipped token).  So this wasn't actually accurate. We now properly get the real offset/width for the skipped token.